src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Categories/Blocks/form.html.twig

```diff
@@ -56,41 +62,62 @@
       {% if categoryForm.id_parent is defined %}
       <div class="form-group row">
         <label class="form-control-label">
-          {{ 'Parent category'|trans }}
+          {{ 'Parent category'|trans({}, 'Admin.Catalog.Feature') }}
         </label>
         <div class="col-sm">
           {{ form_widget(categoryForm.id_parent) }}
         </div>
       </div>
       {% endif %}
 
+      {% if categoryForm.description is defined %}
       <div class="form-group row">
-        {{ ps.label_with_help(('Description'|trans({}, 'Admin.Global')), ('Invalid characters:'|trans({}, 'Admin.Notifications.Info') ~ ' <>;=#{}')) }}
```

Comment:
Is there something wrong with `label_with_help` macro ?

Reply:
nope, its just not needed anymore since help message is moved below input.